packages/navigator/src/browser/navigator-model.ts

```diff
@@ -71,7 +71,9 @@ export class FileNavigatorModel extends FileTreeModel {
 
     protected async createRoot(): Promise<TreeNode | undefined> {
         if (this.workspaceService.opened) {
-            const workspaceNode = WorkspaceNode.createRoot();
```

Comment:
Multi root workspace could have only one root, there should be something like `workpace.multiRoot`

Reply:
I'm not quite sure I follow, what sort of change are you looking for here?

Reply:
see https://github.com/theia-ide/theia/pull/4729/files/2ec33c4360805babfe9fccf31428fd8d46506f29#r269508706, commented on a wrong line, sorry